docs/secure-data-warehouse-on-foundation-3-0-0.md

```diff
@@ -0,0 +1,913 @@
+# How to customize Foundation v3.0.0 for Secured Data Warehouse Blueprint deployment
+
+These instructions explain how to deploy the [Secured Data Warehouse Blueprint](https://github.com/GoogleCloudPlatform/terraform-google-secured-data-warehouse) on top of the [Terraform Example Foundation](https://cloud.google.com/architecture/security-foundations/using-example-terraform) version [v3.0.0](https://github.com/terraform-google-modules/terraform-example-foundation/tree/v3.0.0).
+
+## Overview
+
+The deploy of the Blueprint will use the `production` environment of the business unit 1
+of an existing Terraform Example Foundation and will deploy/create:
+
+- The infrastructure required by the Blueprint (*harness*),
+- The Blueprint itself,
+- A DataFlow reading from a public BigQuery de-identifing data and writing in a non-confidential Bigquery
+- A DataFlow reading from non-confidential Bigquery and re-identifing data and writing in a confidential BigQuery
+- The taxonomy being applied at confidential BigQuery.
+
+The following infrastructure will be created or reused:
+
+- The restricted VPC Service Controls (VPC-SC) perimeter and the shared VPC created in step `3-networks` will be used.
+- Four new project will be created in step `4-projects`:
+  - The Data Ingestion project that will be added to the existing restricted perimeter and shared VPC.
+  - The Non-Confidential Data project that will be added to the existing restricted perimeter and shared VPC.
+  - The Data Governance project that will be added to a new VPC-SC perimeter.
+  - The Confidential Data project that will be added to another new VPC-SC perimeter.
+  - An additional project, outside of any perimeter to host Dataflow template images.
+- The Blueprint infrastructure.
```

Comment:
Isn't `"The Blueprint infrastructure."` too generic for a parent item `"The following infrastructure will be created or reused:" `?
I mean, I didn't fully understand what is expected to be created by reading this item

Reply:
yeah, this should be listed at the main readme, and then add the link here...

Reply:
https://github.com/GoogleCloudPlatform/terraform-google-secured-data-warehouse/pull/373